testing/test_stepwise.py

```diff
@@ -75,6 +75,16 @@ def broken_testdir(testdir):
     return testdir
 
 
+def _strip_resource_warnings(lines):
+    # Assert no output on stderr, except for unreliable ResourceWarnings.
```

Comment:
Can we change this comment? The way it’s written right now is misleading since we don’t assert in the function but after we return.

Alternatively, maybe we could move the assert into the function?

Reply:
Thanks, I updated the comment.

> Alternatively, maybe we could move the assert into the function?

I usually prefer the assert to be in the function itself, that way the stacktrace is nicer.